game/data/groups.py

```diff
@@ -0,0 +1,88 @@
+from enum import Enum
+
+
+class GroupRole(Enum):
+    Unknow = "Unknown"
```

Comment:
Yes the purpose was to have a legacy handling during the migration to the new structure. I initially thought i would migrate it all after the first merge. But i was in a mood of updating all units one day. So i will check if it can be removed completly now.